Sources/Orbit/Components/Icon.swift

```diff
@@ -79,11 +83,11 @@ public extension Icon {
         self.size = size
     }
     
-    /// Creates Orbit Icon component for provided SF Symbol.
+    /// Creates Orbit Icon component for provided SF Symbol with specified color.
     ///
     /// Color is configurable using `foregroundColor` modifier.
-    init(sfSymbol: String, size: Size = .normal) {
-        self.content = .sfSymbol(sfSymbol)
+    init(sfSymbol: String, size: Size = .normal, color: Color? = .inkNormal) {
```

Comment:
Does it have to be optional if it has a default non-nil value?

Reply:
I was thinking about making these shortcut inits with non-nil value, but the fact that they support override colour using `foregroundColor` would be more hidden and limited only to that full verbose init. 

Tried to make the API more similar to `Text` in this regard (although allowing for the override, which is not supported by `Text/Heading` component (yet), as there it may be a bit more limited to support it).

Will update the comments though.

Reply:
That would limit the caller to use the verbose init whenever he would like to override the color using `.foregroundColor()`. As the `Text` component already supports this in a same way, I updated it so both components have more consistent API.

Updated the comments to be more clear and consistent with `Text`.